src/main/java/seedu/address/model/Context.java

```diff
@@ -24,7 +24,7 @@ public Context() {
         this.type = ContextType.LIST_CONTACT;
     }
 
-    private Context(ContextType type) {
+    public Context(ContextType type) {
```

Comment:
This constructor is intended to be private! There's already a static factory method to construct a new `LIST_ACTIVITY` or `LIST_CONTACT` context (they should be named `newListActivityContext` and `newListContactContext` respectively.